packages/nodes-base/nodes/Mattermost/v1/transport/index.ts

```diff
@@ -22,7 +22,7 @@ export interface IAttachment {
 }
 
 /**
- * Make an API request to Telegram
+ * Make an API request to Mattermost
```

Comment:
This is a good opportunity to standardize the removal of JSDoc arg descriptions.